pkg/cvo/sync.go

```diff
@@ -55,6 +62,19 @@ func (optr *Operator) syncUpdatePayload(config *cvv1.CVOConfig, payload *updateP
 	return nil
 }
 
+// getOverrideForManifest returns the override and true when override exists for manifest.
+func getOverrideForManifest(overrides []cvv1.ComponentOverride, manifest lib.Manifest) (cvv1.ComponentOverride, bool) {
+	for idx, ov := range overrides {
+		kind, namespace, name := manifest.GVK.Kind, manifest.Object().GetNamespace(), manifest.Object().GetName()
+		if ov.Kind == kind &&
+			(namespace == "" || ov.Namespace == namespace) && // cluster-scoped objects don't have namespace.
+			ov.Name == name {
+			return overrides[idx], true
```

Comment:
Why `overrides[idx]` instead of `ov`?

Reply:
Don't like using the value from range for anything other than reading....